app/src/main/java/com/example/moviesandroidapp/SplashScreen.kt

```diff
@@ -0,0 +1,28 @@
+package com.example.moviesandroidapp
+
+import android.content.Intent
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.os.Handler
+import android.os.Looper
+import android.widget.TextView
+import com.airbnb.lottie.LottieAnimationView
+
+class SplashScreen : AppCompatActivity() {
```

Comment:
Just for convention, I think you can name this file 'SplashScreenActivity', what you think? 👍